astro/src/content/quickstarts/_5-minute-setup-wizard.mdx

```diff
@@ -4,7 +4,7 @@ Once FusionAuth starts up successfully, you will be taken to the Setup Wizard. T
 
 **Record the email and password** you choose for the administrator account. You won't be able to recover this without setting up an email server, which is beyond the scope of this tutorial.
 
-You can also sign up for the FusionAuth newsletter, which is a great way to stay up-to-date on FusionAuth releases and new features. We will never pitch you, hard-sell or sell your information, so feel free to sign up.
+You can also choose to have the server send a request to sign up for the FusionAuth newsletter, which is a great way to stay up-to-date on FusionAuth releases and new features. We will never pitch you, hard-sell or sell your information, so feel free to sign up.
```

Comment:
```suggestion
You can also choose to sign up for the FusionAuth newsletter, which is a great way to stay up-to-date on FusionAuth releases and new features. We will never pitch you, hard-sell or sell your information, so feel free to sign up.
```

Reply:
I think mentioning the server here is a bit confusing, even though I know we discussed internally.